module/zcommon/zfeature_common.c

```diff
@@ -137,6 +141,22 @@ deps_contains_feature(const spa_feature_t *deps, const spa_feature_t feature)
 	return (B_FALSE);
 }
 
+static boolean_t
+zfs_mod_supported_feature(const char *name)
+{
+#if defined(_KERNEL) || defined(LIB_ZPOOL_BUILD)
```

Comment:
Can you include a comment explaining why this always needs to be true for for `libzpool`.  I assume it's to ensure everything is always enabled for `ztest`, right?

Reply:
see comment above^. libzpool clients (zdb, ztest, etc) will know what is supported (it's what was built).